file.c:4083:47: error: conflicting types for ‘GetLastError’
In file included from /usr/include/w32api/windows.h:70:0,
		 from file.c:18:
/usr/include/w32api/winbase.h:1345:27: note: previous declaration of ‘GetLastError’ was here

--- origsrc/ruby-1.9.3-p392/file.c	2013-02-05 23:14:53.000000000 -0600
+++ src/ruby-1.9.3-p392/file.c	2013-03-14 02:12:28.189558200 -0500
@@ -4080,7 +4080,6 @@ rb_file_truncate(VALUE obj, VALUE len)
 
 #ifdef __CYGWIN__
 #include <winerror.h>
-extern unsigned long __attribute__((stdcall)) GetLastError(void);
 #endif
 
 static VALUE
